src/credentials/tests/CHIPCert_test_vectors.cpp

```diff
@@ -169,6 +169,43 @@ CHIP_ERROR GetTestCertPubkey(uint8_t certType, const uint8_t ** certPubkey, uint
     return err;
 }
 
+CHIP_ERROR GetTestCertAKID(uint8_t certType, ByteSpan & akid)
```

Comment:
Suggest renaming to `GetTestCertExpectedAKID`

Reply:
will do